static/js/controllers/inbox.js

```diff
@@ -467,12 +467,9 @@ var feedback = require('../modules/feedback'),
           controller: 'DeleteDocConfirm',
           model: { docs: docs }
         }).then(function() {
-          if ($state.includes('contacts') || $state.includes('reports')) {
-            if ($scope.selectMode) {
-              $scope.clearSelected();
-            } else {
-              $state.go($state.current.name, { id: null });
-            }
+          if (!$scope.selectMode &&
```

Comment:
Do you think this sounds like something that should be written into the code as a comment? May not be obvious in the future. I'll let you do that in master if you think it's relevant (won't delay the merge).